tools/db/move-team/src/Schema.hs

```diff
@@ -1224,7 +1224,7 @@ importSparUser Env {..} path = do
   if exists
     then do
       putStrLn $ "Importing " <> path <> " to " <> "spar.user"
-      withBinaryFile path ReadMode $ \handle -> do
```

Comment:
I prefer to find hlint stuff in separate PRs (but maybe that was hard because your tooling wouldn't let go of these easily?)

Reply:
Yeah, picking only a few suggestions was hard, and applying all was a good thing, so I applied all in this file.